app/models/program.rb

```diff
@@ -30,6 +30,10 @@ def scheduled
       where.not(start_time: nil).where.not(room: nil)
     end
 
+    def unscheduled
+      confirmed.where('start_time IS NULL OR room_id IS NULL')
```

Comment:
But it is not what we want. As I am using unscheduled in the all events to show then, so unscheduled events should be confirmed